app/controllers/sessions_controller.rb

```diff
@@ -15,7 +15,7 @@ def new
   def create
     user = User.find_by(email: params[:email])
     if user == nil || !user.authenticate(params[:password])
-      flash[:failure] = "Incorrect email or password."
+      flash[:error] = "Incorrect email or password."
```

Comment:
Just curious why you updated this? I might get my answer as I continue to review

Reply:
I checked the css that we were given with the repo and it looks like they already had styling for flash messages of success and error (failure didn't have any default styling). So going forward I think we should try and just stick to `:error` to make use of existing styling.